cmd/env/get.go

```diff
@@ -95,6 +95,42 @@ func NewGetAllEnvironmentsCmd() *cobra.Command {
 
 var ErrUnmarshalling = errors.New("failed to unmarshal environment(s)")
 
+// Converts the `environment` object to [][]string which is used during printing environments as table
+func convertObjToArray(env struct{ environment }) [][]string {
```

Comment:
This function can just take a value of type `environment`, no need to wrap it.
Also, the function needs a better name.

Reply:
I was trying without `struct` but was having some issues. It was failing during the `make build`

Reply:
This is an error thrown by the Go compiler. You can fix it by changing the calls of the function by referencing the `environment` field of the `response.Data` (here it's a reference to the value whose name is the same as its embedded type - this is a less used feature of Go, took me a second to figure this out).

`data = append(data, extractDataForTableOutput(d.environment)...)` and
`data := extractDataForTableOutput(r.Data.environment)`

Reply:
ah good find. pushed up the fix